projects/aca-folder-rules/assets/i18n/en.json

```diff
@@ -21,8 +21,47 @@
         "NO_DESCRIPTION": "No description"
       },
       "ERROR": {
-        "REQUIRED": "This field is required"
-      }
+        "REQUIRED": "This field is required",
+        "RULE_COMPOSITE_CONDITION_INVALID": "One or more condition groups is empty"
+      },
+      "COMPARATORS": {
+        "EQUALS": "(=) Equals",
+        "CONTAINS": "Contains",
+        "STARTS_WITH": "Starts with",
+        "ENDS_WITH": "Ends with",
+        "GREATER_THAN": "(>) Greater than",
+        "LESS_THAN": "(<) Less than",
+        "GREATER_THAN_OR_EQUAL": "(>=) Greater than or equal",
+        "LESS_THAN_OR_EQUAL": "(<=) Less than or equal",
+        "ON": "(=) On",
+        "AFTER": "(>) After",
+        "BEFORE": "(<) Before",
+        "ON_OR_AFTER": "(>=) On or after",
+        "ON_OR_BEFORE": "(<=) On or before",
+        "INSTANCE_OF": "(=) Is"
```

Comment:
It would be good to get support from UA or UX for this one. Lots of things are cm:folders even if they don't seem like it (e.g. a site node is an instance of cm:folder)

Reply:
Not sure about this one yet. I haven't defined any fields to use it so far. I'll likely handle it at the same time as updating the fields for selecting the value